ui/app/templates/partials/status/cluster.hbs

```diff
@@ -132,22 +135,20 @@
           </li>
         </ul>
       </nav>
-      {{#if (has-permission 'metrics' routeParams='requests')}}
+      {{#if ( and (has-permission 'metrics' routeParams='requests') (not cluster.dr.isSecondary))}}
```

Comment:
So I was wrong that this doesn't render on sealed pages - HTTP metrics still shows up if the vault is sealed or un-inititialized. I think there should be a check for an authenticated user here too, then it wouldn't show up on init / sealed pages.

Reply:
good call, i added this check in https://github.com/hashicorp/vault/pull/7997/commits/c469895ed63bb5503e8f4be74cfa5cf5124ff4d7. thanks!